### PR TITLE
docs: add app shell guide

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Add documentation for app shell

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
//cc @jbogarthyde

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -21,7 +21,6 @@
       "title": "Blog"
     }
   ],
```
Comment (alan-agius4) [inline aio/content/navigation.json]: Formatting changes done here are done automatically on save

File: aio/content/guide/app-shell.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Step 2: Create the app shell
+
+<code-example format="." language="bash" linenums="false">
+ng generate app-shell --client-project my-app --universal-project server-app
+</code-example>
+
+`my-app` is the name of your client application server-app is the name of the universal (or server) application.
+
+After running this command you will notice that the angular.json configuration file has been updated. Two new targets were added (and a few other changes):
+
```
Comment (jbogarthyde) [inline aio/content/guide/app-shell.md]: After running this command you will notice that the `angular.json` configuration file has been updated to add two new targets, with a few other changes.

File: aio/content/guide/app-shell.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "browserTarget": "my-app:build",
+    "serverTarget": "my-app:server",
+    "route": "shell"
+  }
+}
+</code-example>
+
+## Step 3: Verify the app is built with the shell content
+Build with the app shell target.
+
```
Comment (jbogarthyde) [inline aio/content/guide/app-shell.md]: Use the CLI  to build the `app-shell` target.

(if we succeed in adding a definition of target to the Glossary, we can make this a link to the term)

File: aio/content/guide/app-shell.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+<code-example format="." language="bash" linenums="false">
+ng run my-app:app-shell
+</code-example>
+
+Verify the build output by openning `dist/my-app/index.html` look for text `app-shell works!` which verifies that the app shell route was rendered as part of the output.
```
Comment (jbogarthyde) [inline aio/content/guide/app-shell.md]: To verify the build output, open `dist/my-app/index.html`. Look for default text `app-shell works!` to show that the app shell route was rendered as part of the output.

**Review by jbogarthyde — COMMENTED** [review]
Editorial suggestions.

**Comment by alan-agius4** [comment]
Thank you so much @jbogarthyde for your review.

PR updated. Can someone add `aio-preview` label please?

Thanks.

**Comment by brandonroberts** [comment]
@alan-agius4 is this ready for merge pending review?

**Comment by alan-agius4** [comment]
Yes it is

**Comment by brandonroberts** [comment]
@alan-agius4 I marked it for merge so that caretaker will see its ready and needs final review.

**Comment by alan-agius4** [comment]
thanks @brandonroberts

**Review by mhevery — APPROVED** [review]

**Pull request closed without merging**